apps/buildog/src/components/user-auth-form.tsx

```diff
@@ -15,10 +16,25 @@ export function UserAuthForm({ className, ...props }: UserAuthFormProps) {
   const router = useRouter();
   async function onSubmit(event: React.SyntheticEvent) {
     event.preventDefault();
-    /*
-      handle login process
-    */
-    router.push("/blog/");
+
+    const email = event.target[0].value;
+    const password = event.target[1].value;
+
+    try {
+      Auth.authenticate({
+        email: email,
+        password: password,
+      });
+
+      router.push("/blog/");
+    } catch (error) {
+      console.log(error);
+      /*
+        Handle login error
+      */
+    }
+    
+    setIsLoading(true);
```

Comment:
additionally solve setIsLoading bug. When I press sign in button with wrong credentials, it stuck I can't repress it because of setIsLoading

And name change will be good for setLoading. 

**setIsLoading -> setLoading
isLoading      -> isLoading**